bchrpc/proxy/middlewares/cors.go

```diff
@@ -0,0 +1,23 @@
+package middlewares
+
+import (
+	"net/http"
+)
+
+// Overrides CORS headers per request.
+func CorsMiddleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// programatically check the origin against a map of allowed origins
+		/*
```

Comment:
Can we remove this commented code block?

Reply:
I removed the code block. our strange runtime router doesn't allow applying middlewares like this.